services/login/src/handlers/mozilla-auth0.js

```diff
@@ -172,10 +157,10 @@ class Handler {
     // array as we do not use Auth0 user linking.
     profile.identities.forEach(({provider, connection}) => {
       if (
-        provider === 'ad' && connection === 'Mozilla-LDAP' ||
+        (provider === 'ad' && connection === 'Mozilla-LDAP') ||
```

Comment:
I think I remember the lint rules at the time complaining about this.  They definitely help humans read the expression!